rmf_task_msgs/srv/GetTask.srv

```diff
@@ -0,0 +1,15 @@
+# Query list of submitted tasks | Get service call
+
+# Whom that call this srv
+string requester
+
+# Input the generated task ID during submission
+# if empty, provide all Submitted Tasks
+string[] task_id
+
+---
+
+bool success
+
+TaskSummary[] active_tasks
+TaskSummary[] terminated_tasks
```

Comment:
How long is a fleet adapter supposed to remember its list of terminated tasks? We should make sure we don't have a memory pile-up for very long running processes.

Reply:
These service calls are meant to be made from UIs and are received by the Dispatcher node. Hence, the dispatcher node is the one storing a list of terminated tasks. I guess we need some kind of timeline for keeping track of such tasks. @tanyouliang95

Reply:
I've opened an issue for this here: https://github.com/osrf/rmf_core/issues/238